GVFS/GVFS.Common/NuGetUpgrade/NuGetFeed.cs

```diff
@@ -128,6 +129,21 @@ public virtual async Task<string> DownloadPackageAsync(PackageIdentity packageId
             return downloadPath;
         }
 
+        public virtual bool VerifyPackage(string packagePath)
+        {
+            VerifyArgs verifyArgs = new VerifyArgs()
+            {
+                Verifications = new VerifyArgs.Verification[] { VerifyArgs.Verification.All },
+                PackagePath = packagePath,
+                CertificateFingerprint = new List<string>() { "3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE" },

```

Comment:
As discussed offline, add a named constant for `3F9001EA83C560D712C24CF213C3D312CB3BFF51EE89435D3430BD06B5D0EECE`